sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -724,11 +730,15 @@ public Key restoreKey(byte[] backup) {
      * call {@link KeyClient#getKey(KeyBase baseKey)} . This will return the {@link Key key} with key material included of its latest version.</p>
      * {@codesnippet com.azure.keyvault.keys.keyclient.listKeys#Context}
      *
+     * <p>It is possible to get full keys with key material from this information. Loop over all the {@link KeyBase key} by page and
```

Comment:
Description is fine, But the header should stay too.
<p><strong>Code Samples to iterate keys by page</strong></p>